.gitignore

```diff
@@ -23,6 +23,7 @@ notebooks/ipfs.log
 .idea/*
 venv/
 test_flask_grid_server.db
+database*
```

Comment:
Q: Should we have this here or add a remove in the tests? (the tests are creating databases which are not deleted)

Reply:
I agree with you. It can be a great new PR.